### ISSUE TITLE
Added charger stop, appears unnecessary

### ISSUE THREAD

**Issue by EVGrokker** [aesculus/EVTO-App-Feedback#377]
v 1.2.0 (15)

>Car Data: X90D, Charger: 48, Wheels: 20, Tires: 0, Pano: 0, Rear CC: 0, Bat Life: 100 
Device Info: iPhone; CPU iPhone OS 10_3_2 like Mac OS X 
Settings: Reserve SoC:20, Default SoC: 90, Region: 0, Units: 0 
Version: 1.2.0 (15) 
Location: Unknown 
Trip: Sand Dollar Inn & Cottages, [7s9m1ofrzeerjucupahwt]

This is a round trip to a destination charging location (Seattle - Moclips WA).

The outbound segment shows no charging stop required, which seems reasonable.

The return segment along the same route includes a supercharging stop at Aberdeen WA, charging from 79% to 79%, which should not be included, since there's no charging delta.

**Comment by aesculus** [comment]
I will now look into this trip to see why it is adding the two charging stops.

**Comment by aesculus** [comment]
Well I just tested this with V1.2 (17) and it works like you would expect. However I want to poke under the covers and make sure it is not coincidental.

**Comment by EVGrokker** [comment]
>Car Data: X90D, Charger: 48, Wheels: 20, Tires: 0, Pano: 0, 5 Seat: 1, Rear CC: 0, Bat Life: 100 
Device Info: iPhone; CPU iPhone OS 10_3_2 like Mac OS X 
Settings: Reserve SoC:20, Default SoC: 90, Region: 0, Units: 0 
Version: 1.2.0 (20) 
IAP: Pro:1, Sub:1 
Location: Unknown 
Trip: Oxford Suites Portland - Jantzen Beach, [r2pehm3pp3pyiu0tcems1]

This trip adds a stop in Centralia but doesn't actually charge. Arrival SoC: 56%, Departure: 56%

**Comment by aesculus** [comment]
I tweaked it a bit in V1.2 (21).  I also tested it with that MS85 with 19" wheels. Last time it worked for an X and then did not add a charger when needed for the MS.

While we are using weather in this case, it appears that depending on the weather we can find that an added charger is not really required. When the weather differs between Seattle and Portland, or more importantly near Centralia, the app sometimes finds that averaging between the two endpoints is not the same as getting forecast for the midpoint.
